src/components/Onboarding/Screens/Apps/VotingSettings/index.js

```diff
@@ -276,15 +240,70 @@ function VotingSettings() {
         value={voteMinAcceptanceQuorum}
         onChange={handleQuorumChange}
       />
-      <div>
-        <Carousel
-          itemWidth={80 * GU}
-          itemHeight={38 * GU}
-          itemSpacing={2 * GU}
-          items={carouselItems}
-          onItemSelected={handleItemSelected}
-        />
-      </div>
+      <DurationFields
+        label={
+          <Fragment>
+            Voting Duration
```

Comment:
Should we update in favor of  `Vote Duration`? Since that´s what we are using in the `Help` component. 

Or we could update the hint for `Voting Duration`